### PR TITLE
get travis working

### PR DESCRIPTION
**Pull request by demmer**
A few changes to update travis now that the repository is open.

### CODE REVIEW

**Comment by demmer** [comment]
@davidvgalbraith @rlgomes take a look

File: .travis.yml [hunk 1]
```diff
@@ -22,9 +22,7 @@ services:
     - elasticsearch
 
 before_script:
-    - git clone https://${GITHUB_TOKEN}:x-oauth-basic@github.com/juttle/juttle.git
-    - pushd juttle ; npm install ; npm link ; popd
+    - npm install juttle@0.1.x
 
 script:
-    - export NODE_PATH="$(dirname `which node`)/../lib/node_modules"
     - mocha
```
Comment (rlgomes) [inline .travis.yml]: In juttle-sql-adapter-common and juttle-graphite-adapter we've started putting the section on scripts into the package.json like so:

https://github.com/juttle/juttle-sql-adapter-common/blob/master/package.json#L10

and then in the travis file you can lint and run the tests through `npm`:

https://github.com/juttle/juttle-sql-adapter-common/blob/master/.travis.yml#L26

**Comment by rlgomes** [comment]
+1 looks fine... just a small suggestion to keep our adapter projects looking more consistent with eachother.

**Comment by demmer** [comment]
That's a much better plan... can you or @davidvgalbraith take it from here?

**Comment by davidvgalbraith** [comment]
I added @rlgomes's fancy linting and made local-mode testing the default. I had to get rid of the testing on node 4.2 as concurrent tests against my single EC2 instance got confused. I'll be able to add it back when I implement `read -index` and `write -index`, at which point I'll be able to hit different indices for the different versions.

**Pull request merged**